src/core/base.js

```diff
@@ -1,18 +1,20 @@
 'use strict';
-const _ = require('lodash');
+
 const sjcl = require('./utils').sjcl;
-const utils = require('./utils');
+const sha256 = require('./utils').sha256;
 const extend = require('extend');
-const convertBase = require('./baseconverter');
 
 const Base = {};
 
 const alphabets = Base.alphabets = {
+  bitcoin: '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz',
   ripple: 'rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz',
-  tipple: 'RPShNAF39wBUDnEGHJKLM4pQrsT7VWXYZ2bcdeCg65jkm8ofqi1tuvaxyz',
-  bitcoin: '123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz'
+  tipple: 'RPShNAF39wBUDnEGHJKLM4pQrsT7VWXYZ2bcdeCg65jkm8ofqi1tuvaxyz'
 };
 
+const codecOptions = {sha256, alphabets, defaultAlphabet: 'ripple'};
```

Comment:
Agreed it would be a pita to instantiate it like this everywhere wanted to use it (even though most ripple clients would just use `Base`).

Anyway, it's Transitional code. We can use ripple-address-codec, already bound to hash.js, instantiated with a bunch of alphabets and using ripple alphabet by default.  The alphabets have symbolic names like `ripple` for `rpshnaf39wBUDNEGHJKLM4PQRST7VWXYZ2bcdeCg65jkm8oFqi1tuvAxyz` and encode/decode functions both honour `alphabet` arg for non default alphabets.